.htaccess

```
@@ -48,7 +50,7 @@ RedirectMatch "/developer/engine/web/(?!faq|history|current-version|\d+\.\d+)(.*
 
 # current-version:15.0
 RewriteCond "%{DOCUMENT_ROOT}/$1/15.0" -d
-RewriteRule "^(.+)/current-version(/.*|$)" "$1/15.0$2"
+RewriteRule "^(.+)/current-version(/.*)?" "$1/15.0$2"
```

```suggestion
RewriteRule "^(.+)/current-version(/.*)?$" "$1/15.0$2"
```

We want to make sure we capture the whole suffix. I think it will anyway given `.*` is greedy by default in PCRE, but including `$` clarifies our intent which helps for future maintenance!

I think you need to copy the same pattern down for the other current-version rules.